src/components/TextBox.tsx

```diff
@@ -70,7 +75,8 @@ export default function TextBox({ field }: TextBoxProps): JSX.Element {
 		<TextBoxContainer>
 			<TextBoxInput
 				style={hideInput}
-				type="number"
+				type="text"
+				pattern="^[0-9]*[.]?[0-9]*$"
```

Comment:
You maybe want to re-use the same regexp at `valid_input_reg_exp.ts` here

Reply:
Added in [85a94e6](https://github.com/ardriveapp/price-calculator/pull/23/commits/85a94e6fd1cf7f18744a7c53437afd6e82b314dc)